pkg/backup/pod_action.go

```diff
@@ -56,12 +56,20 @@ func (a *PodAction) Execute(item runtime.Unstructured, backup *v1.Backup) (runti
 		return nil, nil, errors.WithStack(err)
 	}
 
+	var additionalItems []velero.ResourceIdentifier
+	if pod.Spec.PriorityClassName > "" {
```

Comment:
The `spec.priorityClassName` is a string, suggest to compare it as
```go
if pod.Spec.PriorityClassName != ""
```

Same as below.

Reply:
Corrected.